packages/web/components/Hero/styles.js

```diff
@@ -1,7 +1,13 @@
 import styled from 'styled-components';
 
+const heroImage = () => {
+	const heroImgs = ['1.jpg', '2.jpg', '3.jpg', '4.jpg', '5.jpg', '6.jpg', '7.jpg'];
+	const heroIndexImg = Math.floor(Math.random() * heroImgs.length);
+	return heroImgs[heroIndexImg || 0];
```

Comment:
```suggestion
	return heroImgs[heroIndexImg];
```

Não precisa desse `|| 0`, já que o `Math.floor` vai arredondar pra baixo e o `Math.random` nunca vai retornar 1

Reply:
Boa @luizeboli !